packages/api/cms-api/src/dam/files/files.controller.ts

```diff
@@ -107,6 +107,26 @@ export function createFilesController({ Scope: PassedScope }: { Scope?: Type<Dam
             return this.streamFile(file, res, { range, overrideHeaders: { "Cache-control": "private" } });
         }
 
+        @DisableGlobalGuard()
+        @Get(`/download/preview/${fileUrl}`)
+        async previewDownloadFile(
+            @Param() { fileId }: FileParams,
+            @Res() res: Response,
+            @Headers(CDN_ORIGIN_CHECK_HEADER) cdnOriginCheck: string,
+            @Headers("range") range?: string,
+        ): Promise<void> {
+            this.checkCdnOrigin(cdnOriginCheck);
+
+            const file = await this.filesService.findOneById(fileId);
+
+            if (file === null) {
+                throw new NotFoundException();
+            }
+
+            res.setHeader("Content-Disposition", "attachment");
+            return this.streamFile(file, res, { range });
```

Comment:
@nsams do we need a `cache-control: private` here?

Reply:
yes, everything should be the same as the non-download variant